src/main/java/org/opensrp/domain/postgres/PlanExample.java

```diff
@@ -174,23 +174,23 @@ public boolean isValid() {
 
 		protected void addCriterion(String condition) {
 			if (condition == null) {
-				throw new IllegalArgumentException("Value for condition cannot be null");
+				throw new RuntimeException("Value for condition cannot be null");
```

Comment:
Change all `RuntimeException`  to `IllegalArgumentException`

Reply:
Done